src/service/tables/signatures.h

```diff
@@ -61,9 +61,26 @@ namespace ccf
   using SerialisedMerkleTree =
     ccf::kv::RawCopySerialisedValue<std::vector<uint8_t>>;
 
+  struct CoseSignature
```

Comment:
I don't think we should encode the COSE Signature in base64 in a JSON, I suggest we store it directly as raw value in the map, just like we do for cose_history.